src/utils/Lesson/index.js

```diff
@@ -0,0 +1 @@
+export { getDeploymentAccountIdsFromLesson } from "./utils.js";
```

Comment:
why did you decide to have an `index.js` after exporting directly from `utils.js`?

Reply:
it's just so we can do `import` from "../utils/Lesson"` rather than `import from "../utils/Lesson/utils.js", like how Jason does it`